### DISCUSSION TITLE
Is there a way, via config, to change the "Save" button on all the popup dialogs to "OK" ?

### DISCUSSION THREAD

**Discussion by clarumedia** [tinymce/tinymce#9591] [Ideas]
It can be misleading for users if the editor has been instantiated within a bigger form which has a Save or Update button.

**Comment by clarumedia** [comment]
I managed to achieve it like this, but it's a bit of a hack.

            editor.on('OpenWindow', function(e) {

                //Replace "Save" in any opened dialog with "OK"
                let objDialogFooter=document.querySelector('.tox-dialog__footer');
                let objButtons=objDialogFooter.querySelectorAll('.tox-button');

                for (let i in objButtons){
                    if (objButtons[i].textContent=="Save"){
                        objButtons[i].innerHTML="OK";
                    }
                }
            });

**Comment by TheSpyder** [comment]
We don't have any configs like this. You could try using the translation framework to override the `Save` string, but that would apply everywhere (such as the tooltip of the `save` plugin button)
https://www.tiny.cloud/docs/tinymce/latest/creating-a-plugin/#language-localization

**Reply by clarumedia** [reply]
I'll take a look. Thank you
